test/src/main/java/io/strimzi/test/k8s/Kubernetes.java

```diff
@@ -0,0 +1,314 @@
+                .readingInput(null)
+                .writingOutput(baos)
+                .usingListener(new ExecListener() {
+                    @Override
+                    public void onOpen(Response response) {
+                        LOGGER.info("Reading data...");
+                    }
+
+                    @Override
+                    public void onFailure(Throwable throwable, Response response) {
+                        data.completeExceptionally(throwable);
+                    }
+
+                    @Override
+                    public void onClose(int i, String s) {
+                        data.complete(baos.toString());
+                    }
+                }).exec(command)) {
+            return data.get(1, TimeUnit.MINUTES);
+        } catch (Exception e) {
+            LOGGER.warn("Exception running command {} on pod: {}", command, e.getMessage());
+            return "";
+        }
+    }
+
+    public String execInPodContainer(String podName, String container, String... command) {
+        ByteArrayOutputStream baos = new ByteArrayOutputStream();
+        LOGGER.info("Running command on pod {}: {}", podName, command);
+        CompletableFuture<String> data = new CompletableFuture<>();
+        try (ExecWatch execWatch = CLIENT.inNamespace(namespace).pods()
+                .withName(podName).inContainer(container)
+                .readingInput(null)
+                .writingOutput(baos)
+                .usingListener(new ExecListener() {
+                    @Override
+                    public void onOpen(Response response) {
+                        LOGGER.info("Reading data...");
+                    }
+
+                    @Override
+                    public void onFailure(Throwable throwable, Response response) {
+                        data.completeExceptionally(throwable);
+                    }
+
+                    @Override
+                    public void onClose(int i, String s) {
+                        data.complete(baos.toString());
+                    }
+                }).exec(command)) {
+            return data.get(1, TimeUnit.MINUTES);
+        } catch (Exception e) {
+            LOGGER.warn("Exception running command {} on pod: {}", command, e.getMessage());
+            return "";
+        }
+    }
+
+    public List<Pod> listPods(LabelSelector selector) {
+        return CLIENT.inNamespace(namespace).pods().withLabelSelector(selector).list().getItems();
+    }
+
+    public List<Pod> listPods(Map<String, String> labelSelector) {
+        return CLIENT.inNamespace(namespace).pods().withLabels(labelSelector).list().getItems();
+    }
+
+    public List<Pod> listPods() {
+        return CLIENT.inNamespace(namespace).pods().list().getItems();
+    }
+
+    /**
+     * Gets pod
+     */
+    public Pod getPod(String name) {
+        return CLIENT.inNamespace(namespace).pods().withName(name).get();
+    }
+
+    public Date getPodCreateTimestamp(String podName) {
```

Comment:
`getPodCreationTimestamp()` ? I am not an English master but it make better sense to me with this name.